openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "4.3.1"
+  }
+}
```

Comment:
i need to rebase since these changes went in but wanted to get this out for a looksee